c2nim.nim

```diff
@@ -7,7 +7,7 @@
 #    distribution, for details about the copyright.
 #
 
-import std / [strutils, os, times, parseopt, strscans]
+import std / [strutils, os, times, tempfiles, parseopt, strscans]
```

Comment:
`std/tempfiles` was added to std since 1.6, c2nim needs to keep 1.2.x compatibility.

Reply:
I think it's fine if the _generated_ code is 1.2-compatible (likely a flag controlling the compatibility would be nice) - what c2nim itself uses can be newer